x11-libs/libxcursor/libxcursor-1.1.15.recipe

```diff
@@ -20,7 +20,10 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	lib:libX11$secondaryArchSuffix
+	lib:libx11$secondaryArchSuffix
+	lib:libxau$secondaryArchSuffix
+	lib:libxcb$secondaryArchSuffix
+	lib:libxdmcp$secondaryArchSuffix
```

Comment:
```diff
-	lib:libx11$secondaryArchSuffix
+	lib:libX11$secondaryArchSuffix
-	lib:libxau$secondaryArchSuffix
-	lib:libxcb$secondaryArchSuffix
-	lib:libxdmcp$secondaryArchSuffix
```
(libxcursor does not seem to need `libxau`, `libxcb` or `libxdmcp`.)

For the `libX11` vs `libx11`, I'd keep it as it is currently in master, but I'm OK with your changing it to `libx11` if you wish.

Reply:
Ok, I'm not against it, but having a uniform matter of the libs "provided" would be a good idea then, eg ... for libXau for instance, change provides to "lib:libXau" instead of "lib:libxau" (this is one of many cases for the X11 libs ... imho "requirement" should fit the "provides" ...

Reply:
In my recent changes they require libxau/libxcb/libxdmcp (or should we call them libXau/libXcb/libXdmcp?) :)